packages/flutter/lib/src/rendering/view.dart

```diff
@@ -3,7 +3,7 @@
 // found in the LICENSE file.
 
 import 'dart:io' show Platform;
-import 'dart:ui' as ui show FlutterView, Scene, SceneBuilder, SemanticsUpdate, ViewConstraints;
+import 'dart:ui' as ui show FlutterView, Scene, SceneBuilder, SemanticsUpdate;
```

Comment:
If the `ViewConfiguration` doesn't use `ViewConstraints` anymore, do we need the distinction still? It doesn't appear to be used anywhere in the framework (although it is used in the engine).

Reply:
We still need it for the type used by `FlutterView.physicalConstraints` in the engine. This is similar to how the engine uses ViewPadding that the framework then translates in to EdgeInsets at its earliest convenience because the second one is more expressive.